internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -627,6 +623,15 @@ func setCondition(condition hivev1.ClusterDeploymentCondition, conditions *[]hiv
 	}
 }
 
+func clearClusterAPIErrors(cluster *hivev1.ClusterDeployment) {
```

Comment:
```suggestion
func clearClusterAPIErrors(clusterDeployment *hivev1.ClusterDeployment) {
```

Reply:
nit: lets call that variable clusterDeployment, not to mix it with the db cluster